### PR TITLE
EUI-5088 hide non relevant filter section

### PR DESCRIPTION
**Pull request by scott-turner-hmcts**
### JIRA link ###

https://tools.hmcts.net/jira/browse/EUI-5088

### Change description ###

- Types of work filters are now hidden when the user is on My Cases. They show if on My Tasks / Available Tasks
- My Cases can now be filtered according to service id(s) and location id(s) provided in filters

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

File: test/e2e/features/pageObjects/workAllocation/myWorkPage.js [hunk 1]
```diff
@@ -297,6 +299,25 @@ class MyWorkPage extends TaskList {
     }
 
 
+    async isWorkFilterOfTypeDisplayed(filterType){
+
+        const filetTypeNormalized = filterType.toLowerCase().split(' ').join('');
```
Comment (connorpgpmcelroy) [inline test/e2e/features/pageObjects/workAllocation/myWorkPage.js:304]: Is this supposed to be filterTypeNormalized?

File: test/e2e/features/pageObjects/workAllocation/myWorkPage.js [hunk 2]
```diff
@@ -297,6 +299,25 @@ class MyWorkPage extends TaskList {
     }
 
 
+    async isWorkFilterOfTypeDisplayed(filterType){
+
+        const filetTypeNormalized = filterType.toLowerCase().split(' ').join(''); 
+
+        let filetrContainer = null;
```
Comment (connorpgpmcelroy) [inline test/e2e/features/pageObjects/workAllocation/myWorkPage.js:306]: filterContainer?

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Two typo words are the only issue I can see.

**Review by udaydenduluri33 — APPROVED** [review]
@scott-turner-hmcts Thanks for the changes

File: test/e2e/features/pageObjects/workAllocation/myWorkPage.js (hunk 2 above)
Comment (scott-turner-hmcts) [reply test/e2e/features/pageObjects/workAllocation/myWorkPage.js]: fixed typo

File: test/e2e/features/pageObjects/workAllocation/myWorkPage.js (hunk 1 above)
Comment (scott-turner-hmcts) [reply test/e2e/features/pageObjects/workAllocation/myWorkPage.js]: fixed typo

**Review by connorpgpmcelroy — APPROVED** [review]
Typos resolved - everything else LGTM

**Pull request merged**